doc/conf.py

```diff
@@ -153,7 +153,14 @@
 
 # Sphinx project configuration
 templates_path = ["_templates"]
-exclude_patterns = ["_build", "**.ipynb_checkpoints"]
+exclude_patterns = [
+    "_build",
+    "**.ipynb_checkpoints",
+    # Workaround from https://github.com/executablebooks/MyST-NB/issues/363 to prevent
+    # myst-nb from parsing the .ipynb and .py files generated by sphinx-gallery.
+    "**/*.ipynb",
+    "**/*.py",
```

Comment:
Can these be more specific to point to a specific folder instead of using `**`?

Reply:
Done in 90bb1104c4b7ee7e0f73382232e022647435a685.